port_ocean/ocean.py

```diff
@@ -98,8 +121,16 @@ async def __call__(self, scope: Scope, receive: Receive, send: Send) -> None:
         async def lifecycle(_: FastAPI) -> AsyncIterator[None]:
             try:
                 init_signal_handler()
+                now = datetime.datetime.now()
+                calculation = asyncio.create_task(self.calculate_next_resync(now))
                 await self.integration.start()
                 await self._setup_scheduled_resync()
+                next_resync = await calculation
+                await self.port_client.update_resync_state(
+                    {
+                        "next_resync": next_resync,
+                    }
+                )
```

Comment:
I think this needs be updated right after the resync, instead of as part of the initialization itself, that way we are more consistent and provide actual data.